metrics/server.py

```
@@ -0,0 +1,58 @@
+"""Internal HTTP server for Prometheus metric scraping."""
+
+from __future__ import annotations
+
+import asyncio
```

Flake8 (7.3.0)

[error] 'asyncio' imported but unused

(F401)